src/ontology/mondo-edit.obo

```diff
@@ -243789,8 +243789,10 @@ xref: OMIM:604145 {source="MONDO:equivalentTo", source="DOID:0110430"}
 xref: UMLS:C1858763 {source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="OMIM:604145"}
 is_a: MONDO:0016191 ! qualitative or quantitative defects of titin
 is_a: MONDO:0016333 {source="MONDO:Redundant", source="OMIM:604145"} ! familial dilated cardiomyopathy
+is_a: MONDO:0100494 {source="https://orcid.org/0000-0001-9310-0163"} ! autosomal dominant titinopathy
```

Comment:
Why was this parent term selected vs. `MONDO:0100175` TTN-related myopathy?

Reply:
that is what the user requested

Reply:
oh wait, you are right, I am wrong!